R/splatPop-simulate.R

```diff
@@ -13,6 +13,13 @@
 #' @param gff Either NULL or a data.frame object containing a GFF/GTF file.
 #' @param key Either NULL or a data.frame object containing a full or partial
 #'        splatPop key.
+#' @param eqtl Either NULL or if simulating population parameters directly from
+#'        empirical data, a data.frame with empirical/desired eQTL results.
+#'        To see required format, run `mockEmpiricalSet()` and see eqtl output.
+#' @param means Either NULL or if simulating population parameters directly from
+#'        empirical data, a Matrix of real gene means across a population, where
+#'        each row is a gene and each column is an individual in the population.
+#'        To see required format, run `mockEmpiricalSet()` and see means output.
```

Comment:
It might be possible to build these into the parameters object but let's not worry about it for now.